packages/edit-site/src/components/global-styles/dimensions-panel.js

```diff
@@ -137,10 +186,67 @@ export default function DimensionsPanel( { name } ) {
 		resetPaddingValue();
 		resetMarginValue();
 		resetGapValue();
+		resetContentSizeValue();
+		resetWideSizeValue();
 	};
 
 	return (
 		<ToolsPanel label={ __( 'Dimensions' ) } resetAll={ resetAll }>
+			{ ( showContentSizeControl || showWideSizeControl ) && (
+				<span className="span-columns">
+					Set the width of the main content area.
```

Comment:
Apologies if you're already in the middle of tweaking this, but just a quick drive-by, looks like this needs a `__()` call 🙂

Reply:
Thanks. Fixed.

The dangers of pushing quick diffs from testing. ☠️ 🙂